soccer/gameplay/tactics/coordinated_pass.py

```diff
@@ -67,6 +68,14 @@ def __init__(self,
         self.receiver_required = receiver_required
         self.kicker_required = kicker_required
 
+        self.MIN_BALL_MOVING_SPEED = 0.1
+        self.TOO_CLOSE_TO_FAIL_DISTANCE =0.3
+        self.FAR_DISTANCE = 0.5
+        self.STRICT_CROSS_THRESHOLD = 0.1
+        self.RELAXED_CROSS_THRESHOLD = 0.4
+        self.OPPONENT_DISTANCE_THRESHOLD = 0.27
+        self.BALL_STOPPED_SPEED = 0.05
```

Comment:
We usually make these class variables. I don't know if we still want that, but yeah

Reply:
Yeah, that's probably a better idea.